easybuild/easyconfigs/s/Safetensors/Safetensors-0.4.3-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,301 @@
+easyblock = "CargoPythonBundle"
+
+name = 'Safetensors'
+version = '0.4.3'
+versionsuffix = '-CUDA-%(cudaver)s'
+_rustver = '1.75.0'
+
+homepage = 'https://huggingface.co/docs/safetensors'
+description = """Safetensors is a new simple format for storing tensors safely (as opposed to
+pickle) and that is still fast (zero-copy). Safetensors is really fast.
+
+This variant of Safetensors is installed with support for numpy and PyTorch
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Comment:
would also drop this to gfbf when keeping only SciPy-bundle dep, and drop the CUDA and suffix of course